tc/sgx/trusted_worker_manager/enclave/wpe/work_order_enclave_wpe.cpp

```diff
@@ -70,21 +70,21 @@ tcf_err_t ecall_HandleWorkOrderRequestWPE(const uint8_t* inSerializedRequest,
         wo_processor.ext_work_order_data.clear();
     } catch (tcf::error::Error& e) {
         SAFE_LOG(TCF_LOG_ERROR,
-            "Error in KME(ecall_HandleWorkOrderRequestWPE): %04X -- %s",
+            "Error in KME(ecall_HandleWorkOrderRequest): %04X -- %s",
             e.error_code(), e.what());
         ocall_SetErrorMessage(e.what());
         result = e.error_code();
     } catch (...) {
         SAFE_LOG(TCF_LOG_ERROR,
-            "Unknown error KME(ecall_HandleWorkOrderRequestWPE)");
+            "Unknown error KME(ecall_HandleWorkOrderRequest)");
         result = TCF_ERR_UNKNOWN;
     }
 
     return result;
-}  // ecall_HandleWorkOrderRequestWPE
+}  // ecall_HandleWorkOrderRequest
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-tcf_err_t ecall_GetSerializedResponseWPE(uint8_t* outSerializedResponse,
+tcf_err_t ecall_GetSerializedResponse(uint8_t* outSerializedResponse,
     size_t inSerializedResponseSize) {
```

Comment:
ecall_GetSerializedResponse function definition looks same for all enclave type

Reply:
Yes, they are same. I moved the function to work_order_enclave_common.cpp, a new file.